src/main/resources/Configuration.json

```diff
@@ -1,22 +1,47 @@
 {
+  "ioData": {
+      "inputs": [
+        {"type": "ConsoleInput","name": "console_in"},
+        {"type": "RestInput","name": "rest_in"},
+        {"type": "MatchMaker","name": "match_m"}
+      ],
+      "outputs": [
+        {"type": "ConsoleOutput","name": "console_out"},
+        {"type": "RestOutput","name": "rest_out"},
+        {"type": "MasterUtility","name": "util"},
+        {"type": "AdvantEdgeClient","name": "advant_e"},
+        {"type": "ScenarioDispatcher","name": "scenario_dispatcher"},
+        {"type": "ScenarioEditor","name": "scenario_editor"},
+        {"type": "MatchMaker","name": "match_m"}
+      ],
+      "ports": [
+        {"name": "advant_e", "port" : 10500}
```

Comment:
that looks unnecessarily complex, why not just add the port to the output line? JSON is flexible in this regard.